Content.Server/Salvage/FultonSystem.cs

```diff
@@ -52,7 +52,8 @@ private void Fulton(EntityUid uid, FultonedComponent component)
     {
         if (!Deleted(component.Beacon) &&
             TryComp<TransformComponent>(component.Beacon, out var beaconXform) &&
-            !_container.IsEntityOrParentInContainer(component.Beacon.Value, xform: beaconXform))
+            !_container.IsEntityOrParentInContainer(component.Beacon.Value, xform: beaconXform) &&
+            !Transform(uid).Anchored)
```

Comment:
CanFulton already checks for anchoring, you should always update the "CanX" methods to keep the code clean and not add checks elsewhere.

Reply:
that CanFulton check only happens in SharedFultonSystem when the actual fulton is being placed, which let players anchor the item after placing a fulton on it. the check above happens in the servers FultonSystem right before the item is actually fultoned, which fixes that issue. please help me out if there's a better way of doing that